languages/fsharp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -39,6 +50,9 @@ let toes =
 let multiplyPlusTwo x y =
     let product = x * y
     product + 2
+
+// Trying to access the left, right or product bindings
+// here would result in a compile error
```

Comment:
Nice.

Reply:
Yeah. Your port inspired me to also add some examples of common errors too.